boards/nxp/frdm_imx93/frdm_imx93_mimx9352_a55.dts

```diff
@@ -88,11 +88,34 @@
 	status = "okay";
 };
 
-&lpi2c2 {
+&lpi2c1 {
+	clock-frequency = <I2C_BITRATE_FAST>;
+	pinctrl-0 = <&i2c1_default>;
+	pinctrl-names = "default";
 	status = "okay";
+
+	gpio_exp1: pcal6408@20 {
+		compatible = "nxp,pcal6408";
+		reg = <0x20>;
+		gpio-controller;
+		#gpio-cells = <2>;
+		ngpios = <8>;
+		reset-gpios = <&gpio_exp0 20 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
+		status = "okay";
+		/**
+		 * The reset pin of this device is connected to gpio_exp0, which uses
+		 * the same init level. Must do deferred initialization after reset pin
+		 * is released.
+		 */
+		zephyr,deferred-init;
```

Comment:
As I know it will automatically set initialization sequence according to the dependency as exp0 and exp1 have same init priority but exp1 depends on exp0, so exp0 should be initialized firstly, then exp1 will be initialized, so no need to use deffered-init